hack/kind/calico-ipv6/setup-kind.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+set -xv
+
+CLUSTER=${CLUSTER:-netpol-calico-ipv6}
+
+
+kind create cluster --name "$CLUSTER" --config kind-config.yaml
+until kubectl cluster-info;  do
+    echo "$(date)waiting for cluster..."
+    sleep 2
+done
+
+
+kubectl get nodes
+kubectl get all -A
+
+kubectl apply -f calico.yaml
+# was: had to add 2 entries to calico configmap:
+#   https://docs.projectcalico.org/networking/ipv6#enable-ipv6-only
+#kubectl apply -f https://docs.projectcalico.org/manifests/calico.yaml
+
+kubectl -n kube-system set env daemonset/calico-node FELIX_XDPENABLED=false
+kubectl -n kube-system set env daemonset/calico-node FELIX_IPV6SUPPORT=true
```

Comment:
You could just make this change in calico.yaml now

Reply:
👍 awesome!  looks like this is working on my mac now 😄 

saving a copy of the calico yaml seems like the easiest approach, is that a horrible idea?

really appreciate the help, 🙇

Reply:
> saving a copy of the calico yaml seems like the easiest approach, is that a horrible idea?

It'll work, but it means you'll need to raise a PR every time Calico does a release to get the latest.  

I've recently been using kustomize for stuff - it might be more elegant here to:
- create a dir to hold the yaml and kustomize config - let's call it `kustom`
- download `calico.yaml` into `kustom/calico.yaml`
- create a `kustom/kustomize.yaml` file 
- use `kubectl apply -k kustom` to modify and apply it in one step

Reply:
Gotcha, thanks for the suggestion!  Opened #109 to track this 😄